### PR TITLE
add capistrano plugin to index

### PR DESCRIPTION
**Pull request by glaszig**

### CODE REVIEW

File: plugins/capistrano/content.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - name: deploy production
+    image: glaszig/drone-capistrano:2.7.4
+    settings:
+      tasks: production deploy deploy:restart
+    environment:
+      BUNDLE_PATH: vendor/bundle
+      CAPISTRANO_PRIVATE_KEY:
+        from_secret: production_private_key
+      CAPISTRANO_PUBLIC_KEY:
+        from_secret: production_public_key
```
Comment (d1wilko) [inline plugins/capistrano/content.yaml:25]: can these go under settings instead of environment?

**Review by d1wilko — CHANGES_REQUESTED** [review]
coming together nicely - just 2 questions

File: plugins/capistrano/original.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
```
[the remark this answers is not in the document]
Comment (glaszig) [reply plugins/capistrano/original.md:1]: fixed.

File: plugins/capistrano/content.yaml (hunk 1 above)
Comment (glaszig) [reply plugins/capistrano/content.yaml]: yes.

**Review by d1wilko — APPROVED** [review]
LGTM :)

**Pull request merged**